OWNERS_ALIASES

```diff
@@ -221,6 +221,7 @@ aliases:
     - fejta
     - krzyzacy
     - ixdy
+    - justaugustus
```

Comment:
@BenTheElder @spiffxp

Reply:
have you looked at where this is used?
I feel like the e2e framework is not the sort of thing you're typically engaged in.

Reply:
> have you looked at where this is used?

Yep!
- `hack/jenkins/OWNERS`
- `test/e2e/framework/OWNERS`

> I feel like the e2e framework is not the sort of thing you're typically engaged in.

...but I could be! I have a general interest in helping out in places where this alias will be used.

Reply:
I sort of want to delete this alias rather than anticipate its usage in more places.  You are a reviewer over hack/jenkins via a different OWNERS file now.  So getting in here really would be equivalent to test/e2e/framework/OWNERS (which I could directly approve, instead of us having to get a root approver).

My concern is lack of authorship or prior reviews in that area, given the general requirement of [history of review and authorship in a subproject](https://github.com/kubernetes/community/blob/master/community-membership.md#community-membership) or [the more detailed requirements](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-1)
```
out of 400 most recently updated 'merged' PRs in 'kubernetes/kubernetes' involving 'justaugustus'...
11 touched files matching regex 'test/e2e/framework' and no files matching '^$'
of those, justaugustus authored 0, and commented '/lgtm' on 1
```